src/BlockTypes/OrderConfirmation/BillingWrapper.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes\OrderConfirmation;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+use Automattic\WooCommerce\Blocks\BlockTypes\OrderConfirmation\BillingAddress;
+
+/**
+ * BillingWrapper class.
+ */
+class BillingWrapper extends AbstractOrderConfirmationBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'order-confirmation-billing-wrapper';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return string | void Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		$order = $this->get_order();
+
+		if ( ! $order || ! $this->is_current_customer_order( $order ) || ! $order->has_billing_address() ) {
+			return '';
+		}
+
+		$classname          = $attributes['className'] ?? '';
+		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
+
+		if ( isset( $attributes['align'] ) ) {
+			$classname .= " align{$attributes['align']}";
+		}
+
+		return sprintf(
+			'<div class="wc-block-%4$s %1$s %2$s">%3$s</div>',
+			esc_attr( $classes_and_styles['classes'] ),
+			esc_attr( $classname ),
+			$content,
```

Comment:
@mikejolley, I ended up using the provided `$content` attribute directly to render the `Billing Address` Inner Block.  I initially used the following logic to get the Inner Blocks:

```PHP
$inner_blocks_html = ''; 
 foreach ( $block->inner_blocks as $inner_block ) { 
 	$inner_blocks_html .= $inner_block->render(); 
 } 

$content = $inner_blocks_html;
```

So, I'm not certain which approach is the best!